.circleci/config.yml

```diff
@@ -144,13 +135,10 @@ jobs:
     executor: executor_med
     steps:
       - prepare
-      - setup_remote_docker
+      - setup_remote_docker:
+          version: 20.10.11
       - attach_workspace:
           at: ~/project
-      - run:
```

Comment:
Any reason for removing hadolint?

Reply:
hadolist forces to pin numbers to components downloaded from `apt get...`, which we don't want to in order to get the latest version without any known vulnerabilities.